modules/virt-creating-linux-bridge-nncp.adoc

```diff
@@ -46,3 +46,15 @@ spec:
 <6> Disables IPv4 in this example.
 <7> Disables STP in this example.
 <8> The node NIC to which the bridge is attached.
+
+[NOTE]
+====
+To create the NNCP manifest for a Linux bridge using OSA with {ibm-z-name}, you must disable VLAN filtering by the setting the `rx-vlan-filter` to `false` in the `NodeNetworkConfigurationPolicy` manifest.
+
+Alternatively, if you have SSH access to the node, you can disable VLAN filtering by running the following command:
+
+[source,terminal]
+----
+sudo ethtool -K <osa-interface-name> rxvlan off
```

Comment:
Super tiny nit, but this command is missing the prompt `$`. If you don't mind, please fix, but if necessary, I would still merge this PR as-is